features/headings.yml

```diff
@@ -1,6 +1,5 @@
-draft_date: 2024-06-05
-name: <h1>
-description: The `<h1>` element TODO.
+name: Section headings elements
```

Comment:
I'm unsure about naming for collections of elements. For something like `<picture>` and `<table>` I think we should name the feature after those elements, even though other elements are involved. Here, just`<h1>` doesn't represent all elements. But how about "&lt;h1>-&lt;h6>"? @ddbeck WDYT?

Reply:
done